package.json

```diff
@@ -1,40 +1,36 @@
 {
   "name": "witches",
-  "version": "1.0.0",
```

Comment:
Should we have a version?

Reply:
i don't think it is necessary for nuxt configuration but if it's useful for organisation or version control I can add it in? What version number should this be?

Reply:
[Read up on this](https://docs.npmjs.com/cli/v7/configuring-npm/package-json#version) and the version is optional. We probably do want to give this a 2.0.0 version at some point but we won't publish on npm so I think it is ok to leave this.